img/style.less

```diff
@@ -1218,8 +1218,9 @@ button.lms-ui-button {
 	.fa-icon;
 	.fas;
 	.lms-ui-icon;
+	color: darkorange;
```

Comment:
ale tu zmieniać? Chyba raczej to powinno trafić do oddzielnej klasy:
.lms-ui-menu-icon {
  .lms-ui-users-icon;
  color: darkorange;
}
i dodatkowo definicja klasy lms-ui-users-icon.
Fajnie by było, żebyśmy mieli definicje ikon niezależne od kolorów, a potem w różnym kontekście dodawane efekty wizualne, np. zmiana koloru.
Więc to powinno być jakby dziedziczenie z dwóch klas .lms-ui-users-icon i .lms-ui-menu-icon (być może, a może trzeba poprawić .lms-ui-fixedwidth-icon?

Reply:
> Fajnie by było, żebyśmy mieli definicje ikon niezależne od kolorów

Czy jest to sens rozdrabniać na tak małe klasy? Ja zakładam, że w całym LMS wszystkie ikony są takie same czyli: lms-ui-customer-icon ma identyczny kolor jak i wygląd w całym LMS, jeśli chcesz zmienić w jakimś zastosowaniu/kontekscie to zawsze możesz zrobić dziedziczenie całej klasy z nadpisaną tą właściwością.

Wszystkie dodatkowe właściwości (przesunięcie, zmieniona wielkość) są w innych klasach tutaj zrobiłem np. lms-ui-menu-icon oraz lms-ui-quicksearch-icon, są to czynności powtarzalne i wynikające z zastosowania ikonki w danym miejscu. Więc nie można ich zdefiniować w klasach ikon.

Ale kolor? Jest sens pakować to do oddzielnej klasy?

Reply:
Definicje właściwości z klasy lms-ui-fixedwidth-icon proponuję przenieść do lms-ui-menu-icon. Wydaje mi się że raczej nigdzie indziej nie będzie potrzebna ta klasa.